test/lib/classify/cache_test.rb

```diff
@@ -5,9 +5,60 @@
 class PrimerClassifyCacheTest < Minitest::Test
   def test_clear_clears_the_cache
     Primer::Classify::Cache.instance.clear!
+    Primer::Classify::AttrCache.instance.clear!
 
-    assert_empty Primer::Classify::Cache.instance.instance_variable_get(:@lookup)
+    assert Primer::Classify::Cache.instance.empty?
+    assert Primer::Classify::AttrCache.instance.empty?
   ensure
-    Primer::Classify::Cache.instance.preload!
+    Primer::Classify::AttrCache.instance.preload!
+  end
+
+  def test_evicts_entries
```

Comment:
```suggestion
  def test_evicts_lru_entries
```

I saw what I believed to be some LRU behavior in the code above, what are your thoughts on naming the test to reflect that we expect the cache to behave that way?

Reply:
I like it!